WooCommerce/Classes/ViewRelated/Dashboard/MyStore/Cells/ProductTableViewCell.xib

```diff
@@ -63,19 +64,19 @@
                     </stackView>
                     <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="lOL-Ke-vhW">
                         <rect key="frame" x="61" y="103.5" width="227" height="0.5"/>
-                        <color key="backgroundColor" systemColor="separatorColor" red="0.23529411759999999" green="0.23529411759999999" blue="0.26274509800000001" alpha="0.28999999999999998" colorSpace="custom" customColorSpace="sRGB"/>
+                        <viewLayoutGuide key="safeArea" id="dER-lG-WQT"/>
+                        <color key="backgroundColor" systemColor="separatorColor"/>
                         <constraints>
                             <constraint firstAttribute="height" constant="0.5" id="XW4-X6-mV7"/>
                         </constraints>
-                        <viewLayoutGuide key="safeArea" id="dER-lG-WQT"/>
                     </view>
                 </subviews>
                 <constraints>
                     <constraint firstAttribute="trailing" secondItem="lOL-Ke-vhW" secondAttribute="trailing" id="5MW-g7-JV0"/>
                     <constraint firstAttribute="trailing" secondItem="VJh-YT-pnW" secondAttribute="trailing" constant="16" id="EuK-Sc-fLf"/>
```

Comment:
I tried to use system margins on a dashboard screen and learned that it is a more complex task and decided left it out of this PR. Because dashboard is created from a different components added to a scroll view and another tab / segment bar it is needed to do more work to propagate margins to all the views.
I've created and issue to fix it later: #3413

Reply:
ah interesting, Dashboard is a bit different from other use cases. we'll handle it separately in the issue then 👍